src/cldfofflinebrowser/commands/create.py

```diff
@@ -97,6 +98,8 @@ def run(args):
         for c in ['Latitude', 'Longitude']:
             if c in p:
                 del p[c]
+        if not p['latitude'] or not p['longitude']:
```

Comment:
This should explicitly compare against `None`, i.e. `if p['latitude'] is None ...`. Otherwise any 0 coordinates will be excluded.

Reply:
Arg - correct - done